RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
+
+Fix a bug "a custom resolver with permission_classes doesn't accept @sync_to_async decorator"
+The root cause was inspect.iscoroutinefunction() function returns True only for functions defined with "async def".
```

Comment:
Probably this needs to be updated after the change I requested? As this is still going to be used for python 3.12+ (but it actually works in there)